simexpal/build.py

```diff
@@ -65,7 +65,7 @@ class Phase(IntEnum):
 	COMPILE = 4
 	INSTALL = 5
 
-def make_build_in_order(cfg, build, wanted_builds, wanted_phases):
+def make_build_in_order(args, cfg, build, wanted_builds, wanted_phases):
```

Comment:
Make a flag out of `args.f` so you don't pass in this external dependency.

Reply:
Done. Instead of `args.f`, only the `--reclone` flag (`args.reclone` is passed into the functions, with a default value of `False`)